src/tools/qt-gui/README.md

```diff
@@ -1,14 +1,35 @@
-This is a technical preview of a new gui for Elektra.
+# Elektra GUI Documentation #
 
-To build it, make sure that you have Qt5 cmake files in your
-CMAKE PREFIX PATH and that you have added qt-gui in TOOLS.
-You can can also use ALL (will add all tools):
+## Introduction ##
 
-    cmake -DTOOLS=ALL -DCMAKE_PREFIX_PATH=/opt/Qt5.3.0/5.3/gcc_64/lib/cmake .
+This document aims to provide information about Elektra's new `qt-gui` feature. `qt-gui` offers a graphical
+interface for users of Elektra. It allows users to create, manage, edit, and delete keys stored in KDB.
 
+## Compiling and Installation ##
 
-Dependencies: Qt5.3 and libdrm-dev
+### Dependencies ###
 
-If you have wheezy-backports enabled and do not use kde4, you can also install:
+In order to compile and use the new `qt-gui` there are a few dependencies which must be installed. 
 
-    apt-get install qtdeclarative5-dev
+- qt5.3 or greater
+- libdrm-dev
+
+I was able to install the correct dependencies on KDE using the command:	
+`sudo apt-get install qt5-default qml-module-qtquick-controls qml-module-qtquick-dialogs qml-module-qtquick-layouts qml-module-qtgraphicaleffects libdrm-dev`
+
+Note: If you install qt5 manually, make sure the qt5 libarires are inlcude in `LD_LIBRARY`.
```

Comment:
typos "libarires inlcude"

I think more -DCMAKE_PREFIX_PATH=/opt/Qt5.3.0/5.3/gcc_64/lib/cmake  is the important piece. I dont think LD_LIBRARY_PATH is necessary, you mistyped it and it still worked ;)